<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="9" passed="1" failed="6" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-27T16:57:10 IST" name="Surefire suite" finished-at="2024-12-27T16:58:52 IST" duration-ms="101690">
    <groups>
      <group name="regressionTest">
        <method signature="createOragnizationtc_01Test.createOragnizationtc_02withIndustriesTest()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" name="createOragnizationtc_02withIndustriesTest" class="Module_generic_Org_test.createOragnizationtc_01Test"/>
        <method signature="createOragnizationtc_01Test.createorganizationtc_03WithPhoneNumberTest()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" name="createorganizationtc_03WithPhoneNumberTest" class="Module_generic_Org_test.createOragnizationtc_01Test"/>
        <method signature="createOragnizationtc_01Test.deleteOrgTest()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" name="deleteOrgTest" class="Module_generic_Org_test.createOragnizationtc_01Test"/>
        <method signature="createContact_tc_01Test.createContact_tc_02withSupportDateTest()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" name="createContact_tc_02withSupportDateTest" class="Modulegeneric_Contact_test.createContact_tc_01Test"/>
        <method signature="createContact_tc_01Test.createContact_tc_03_withOrgTest()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" name="createContact_tc_03_withOrgTest" class="Modulegeneric_Contact_test.createContact_tc_01Test"/>
      </group> <!-- regressionTest -->
      <group name="smokeTest">
        <method signature="createOragnizationtc_01Test.createOragnizationtc_01Test()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" name="createOragnizationtc_01Test" class="Module_generic_Org_test.createOragnizationtc_01Test"/>
        <method signature="createContact_tc_01Test.createContact_tc_01Test()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" name="createContact_tc_01Test" class="Modulegeneric_Contact_test.createContact_tc_01Test"/>
      </group> <!-- smokeTest -->
    </groups>
    <test started-at="2024-12-27T16:57:10 IST" name="Surefire test" finished-at="2024-12-27T16:58:52 IST" duration-ms="101690">
      <class name="Modulegeneric_Contact_test.createContact_tc_01Test">
        <test-method is-config="true" signature="bs()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:10 IST" name="bs" finished-at="2024-12-27T16:57:10 IST" duration-ms="34" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bs -->
        <test-method is-config="true" signature="bc()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:11 IST" name="bc" finished-at="2024-12-27T16:57:15 IST" duration-ms="4247" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bc -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:15 IST" name="bm" finished-at="2024-12-27T16:57:24 IST" duration-ms="9197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method signature="createContact_tc_01Test()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:24 IST" name="createContact_tc_01Test" finished-at="2024-12-27T16:57:27 IST" duration-ms="3289" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.practice.gen_crm_WebDriverutility.utilityclassobject.getTest()" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.practice.gen_crm_WebDriverutility.utilityclassobject.getTest()" is null
at Modulegeneric_Contact_test.createContact_tc_01Test.createContact_tc_01Test(createContact_tc_01Test.java:55)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createContact_tc_01Test -->
        <test-method is-config="true" signature="am()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:27 IST" name="am" finished-at="2024-12-27T16:57:30 IST" duration-ms="2293" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:30 IST" name="bm" finished-at="2024-12-27T16:57:34 IST" duration-ms="3960" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method signature="createContact_tc_02withSupportDateTest()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:34 IST" name="createContact_tc_02withSupportDateTest" finished-at="2024-12-27T16:57:34 IST" duration-ms="248" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.practice.gen_crm_WebDriverutility.utilityclassobject.getTest()" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.practice.gen_crm_WebDriverutility.utilityclassobject.getTest()" is null
at Modulegeneric_Contact_test.createContact_tc_01Test.createContact_tc_02withSupportDateTest(createContact_tc_01Test.java:106)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createContact_tc_02withSupportDateTest -->
        <test-method is-config="true" signature="am()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:34 IST" name="am" finished-at="2024-12-27T16:57:40 IST" duration-ms="5814" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:40 IST" name="bm" finished-at="2024-12-27T16:57:44 IST" duration-ms="3895" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method signature="createContact_tc_03_withOrgTest()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:44 IST" name="createContact_tc_03_withOrgTest" finished-at="2024-12-27T16:57:44 IST" duration-ms="647" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.practice.gen_crm_WebDriverutility.utilityclassobject.getTest()" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.practice.gen_crm_WebDriverutility.utilityclassobject.getTest()" is null
at Modulegeneric_Contact_test.createContact_tc_01Test.createContact_tc_03_withOrgTest(createContact_tc_01Test.java:179)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createContact_tc_03_withOrgTest -->
        <test-method is-config="true" signature="am()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:44 IST" name="am" finished-at="2024-12-27T16:57:50 IST" duration-ms="5409" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="ac()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:57:50 IST" name="ac" finished-at="2024-12-27T16:57:50 IST" duration-ms="483" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ac -->
        <test-method is-config="true" signature="as()[pri:0, instance:Modulegeneric_Contact_test.createContact_tc_01Test@4386f16]" started-at="2024-12-27T16:58:52 IST" name="as" finished-at="2024-12-27T16:58:52 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- as -->
      </class> <!-- Modulegeneric_Contact_test.createContact_tc_01Test -->
      <class name="Module_generic_Org_test.createOragnizationtc_01Test">
        <test-method is-config="true" signature="bc()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:57:50 IST" name="bc" finished-at="2024-12-27T16:57:53 IST" duration-ms="2737" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bc -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:57:53 IST" name="bm" finished-at="2024-12-27T16:58:02 IST" duration-ms="9615" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method signature="createOragnizationtc_01Test()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:02 IST" name="createOragnizationtc_01Test" finished-at="2024-12-27T16:58:02 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.practice.gen_crm_WebDriverutility.utilityclassobject.getTest()" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.practice.gen_crm_WebDriverutility.utilityclassobject.getTest()" is null
at Module_generic_Org_test.createOragnizationtc_01Test.createOragnizationtc_01Test(createOragnizationtc_01Test.java:48)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createOragnizationtc_01Test -->
        <test-method is-config="true" signature="am()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:02 IST" name="am" finished-at="2024-12-27T16:58:08 IST" duration-ms="5761" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:08 IST" name="bm" finished-at="2024-12-27T16:58:12 IST" duration-ms="3467" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method signature="createOragnizationtc_02withIndustriesTest()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:12 IST" name="createOragnizationtc_02withIndustriesTest" finished-at="2024-12-27T16:58:12 IST" duration-ms="101" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because "com.practice.list_crm_Listnerutility.ListnerImple.test" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because "com.practice.list_crm_Listnerutility.ListnerImple.test" is null
at Module_generic_Org_test.createOragnizationtc_01Test.createOragnizationtc_02withIndustriesTest(createOragnizationtc_01Test.java:83)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createOragnizationtc_02withIndustriesTest -->
        <test-method is-config="true" signature="am()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:12 IST" name="am" finished-at="2024-12-27T16:58:17 IST" duration-ms="5010" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:17 IST" name="bm" finished-at="2024-12-27T16:58:21 IST" duration-ms="4115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method signature="createorganizationtc_03WithPhoneNumberTest()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:21 IST" name="createorganizationtc_03WithPhoneNumberTest" finished-at="2024-12-27T16:58:21 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because "com.practice.list_crm_Listnerutility.ListnerImple.test" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because "com.practice.list_crm_Listnerutility.ListnerImple.test" is null
at Module_generic_Org_test.createOragnizationtc_01Test.createorganizationtc_03WithPhoneNumberTest(createOragnizationtc_01Test.java:121)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createorganizationtc_03WithPhoneNumberTest -->
        <test-method is-config="true" signature="am()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:21 IST" name="am" finished-at="2024-12-27T16:58:26 IST" duration-ms="5503" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:26 IST" name="bm" finished-at="2024-12-27T16:58:30 IST" duration-ms="3414" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method signature="deleteOrgTest()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:30 IST" name="deleteOrgTest" finished-at="2024-12-27T16:58:48 IST" duration-ms="17709" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteOrgTest -->
        <test-method is-config="true" signature="am()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:48 IST" name="am" finished-at="2024-12-27T16:58:52 IST" duration-ms="4174" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="ac()[pri:0, instance:Module_generic_Org_test.createOragnizationtc_01Test@1184ab05]" started-at="2024-12-27T16:58:52 IST" name="ac" finished-at="2024-12-27T16:58:52 IST" duration-ms="471" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ac -->
      </class> <!-- Module_generic_Org_test.createOragnizationtc_01Test -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
